osu.Game.Rulesets.Osu/Mods/OsuModFlashlight.cs

```diff
@@ -63,6 +63,7 @@ public OsuFlashlight(OsuModFlashlight modFlashlight)
                 followDelay = modFlashlight.FollowDelay.Value;
 
                 FlashlightSize = new Vector2(0, GetSizeFor(0));
+                FlashlightSmoothness = 1.4f;
```

Comment:
osu!catch at very least also seems to use a similar smoothness:

![Parallels Desktop 2022-10-03 at 12 23 18](https://user-images.githubusercontent.com/191335/193575680-665ed559-54d1-459a-a264-d1fa0f48d9ef.png)

Did you want to look at applying it to other rulesets here or as a separate effort?

Reply:
Done in the following commits.